provider/aws/cache.go

```diff
@@ -4,14 +4,53 @@ import (
 	"fmt"
 
 	"github.com/convox/praxis/types"
+	"github.com/go-redis/redis"
 )
 
-var cacheBuffer = make(map[string]interface{})
-
 func (p *Provider) CacheFetch(app, cache, key string) (map[string]string, error) {
-	return nil, nil
+	client, err := p.redisClient(app, cache)
+	if err != nil {
+		return nil, err
+	}
+
+	return client.HGetAll(fmt.Sprintf("%s:%s:%s", app, cache, key)).Result()
 }
 
 func (p *Provider) CacheStore(app, cache, key string, attrs map[string]string, opts types.CacheStoreOptions) error {
-	return fmt.Errorf("unimplemented")
+	client, err := p.redisClient(app, cache)
+	if err != nil {
+		return err
+	}
+
+	data := map[string]interface{}{}
+	for k, v := range attrs {
+		data[k] = v
+	}
+
+	return client.HMSet(fmt.Sprintf("%s:%s:%s", app, cache, key), data).Err()
+}
+
+func (p *Provider) redisClient(app, cache string) (*redis.Client, error) {
+	re, err := p.appOutput(app, fmt.Sprintf("Cache%sEndpoint", upperName(cache)))
```

Comment:
There's a physical ID resource we could use to make an API call and get the endpoint. In turn caching that.